watchtower/wtpolicy/policy.go

```diff
@@ -200,8 +200,8 @@ func (p *Policy) ComputeRewardOutputs(totalAmt btcutil.Amount,
 
 	// TODO(conner): replace w/ configurable dust limit
 	// Check that the created outputs won't be dusty. The sweep pkscript is
-	// currently a p2wpkh, so we'll use that script's dust limit.
-	if sweepAmt < lnwallet.DustLimitForSize(input.P2WPKHSize) {
+	// currently a p2tr, so we'll use that script's dust limit.
+	if sweepAmt < lnwallet.DustLimitForSize(input.P2TRSize) {
```

Comment:
Again, not sure if this is possible here, but if this is an older watchtower that uses the p2wkh reward script, it would have a lower dust limit of 294 rather than 330 and wouldn't get included in some cases. Could be fixed by parsing

Reply:
Switched to parse err thang

Reply:
Actually went to parse everything but then realized that we're just passing in the script size here, which we already know (based on the length of the script). So either way it's properly dynamic now.